packages/open-next/src/core/util.ts

```diff
@@ -28,7 +28,11 @@ import {
 overrideNextjsRequireHooks(NextConfig);
 applyNextjsRequireHooksOverride();
 //#endOverride
+
+//#override cacheHandlerPath
 const cacheHandlerPath = require.resolve("./cache.cjs");
```

Comment:
After some thought, i don't see how setting the `NEXT_CACHE_HANDLER_PATH` env would benefit us that much, at least not here. 
The only way this would be useful would be at build time so that the cache is available during `next build`, but then it would mean that we'll have to either include open-next in the bundle (or at least a precompiled version of the cache in the `node_modules`) or still override it at build time like we do today.
It also means that both the tag and incremental cache would need to be available at build time

Reply:
Here again we can not delete this but we should override or it will break the build.